README.md

```diff
@@ -6,7 +6,8 @@
 
 ## Introduction
 
-Adds the Pipeline `build` step, which triggers builds of other jobs.
+Adds the Pipeline `build` step, which triggers builds of other jobs. Use the [_"Pipeline Syntax" Snippet Generator_](https://jenkins.io/redirect/pipeline-snippet-generator) to get a detailed example for your Build.
+The Pipeline Syntax snippet generator helps the user generate steps for jenkins pipeline.
```

Comment:
Thanks for the submission.  We generally prefer "sentence per line" so that reviews and changes are easier.
```suggestion
Adds the Pipeline `build` step, which triggers builds of other jobs.
Use the [_"Pipeline Syntax" Snippet Generator_](https://jenkins.io/redirect/pipeline-snippet-generator) to get a detailed example for your build step.
The Pipeline Syntax snippet generator helps the user generate steps for Jenkins pipeline.
```

Reply:
Alright, that's well noted. Thanks @MarkEWaite